roles/openshift_cloud_provider/tasks/vsphere.yml

```diff
@@ -3,4 +3,26 @@
   template:
     dest: "{{ openshift.common.config_base }}/cloudprovider/vsphere.conf"
     src: vsphere.conf.j2
-  when: openshift_cloudprovider_vsphere_username is defined and openshift_cloudprovider_vsphere_password is defined and openshift_cloudprovider_vsphere_host is defined and openshift_cloudprovider_vsphere_datacenter is defined and openshift_cloudprovider_vsphere_datastore is defined
+  when: 
+  - openshift_cloudprovider_vsphere_username is defined 
+  - openshift_cloudprovider_vsphere_password is defined 
+  - openshift_cloudprovider_vsphere_host is defined 
+  - openshift_cloudprovider_vsphere_datacenter is defined 
+  - openshift_cloudprovider_vsphere_datastore is defined
+
+- name: configure vsphere svc account
+  include_role:
+    name: openshift_cloud_provider
+    tasks_from: vsphere-svc
+  when:
+  - openshift_cloudprovider_kind is defined
+  - openshift_cloudprovider_kind == 'vsphere'
+  - openshift_version | version_compare('3.9', '>=')
```

Comment:
I don't think this is needed - `master` branch won't be able to install 3.9 anyway

Reply:
The svc account change is only applicable to versions above 3.9. Shouldnt we check for it to be on the safe side?